bundle/02_mednist_classification.ipynb

```diff
@@ -265,7 +265,9 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        device: '@device'
</pre><p>The indentation here now is incorrect, these lines that should be under <em>target</em> should also be moved left by 2 spaces.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1618/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>